src/every.test.ts

```diff
@@ -0,0 +1,107 @@
+import { every } from "./every";
+import { isNumber } from "./isNumber";
+import { pipe } from "./pipe";
+
+describe("runtime", () => {
+  describe("data-first", () => {
+    test("returns correct values on simple arrays", () => {
+      const trueInput = [1, 2, 3];
+      const trueData = every(trueInput, (val) => val < 5);
+      expect(trueData).toBe(true);
+
+      const falseInput = ["foo", "bar", "buzz"];
+      const falseData = every(falseInput, (val) => val.length > 3);
+      expect(falseData).toBe(false);
+    });
```

Comment:
Keep tests simple and short; it's better to have more tests than to have tests do more; that way, when a test fails, the actual failure is reported. And we can debug the specific case instead of having to comment out non-failing parts.
```suggestion
    test("all values meet the criteria", () => {
      expect(every([1, 2, 3], (val) => val < 5)).toBe(true);
    });
    
    test("a value doesn't meet  the criteria", () => {
      expect(every(["foo", "bar", "buzz"], (val) => val.length > 3)).toBe(false);
    });
```